spec/system/candidate_interface/carry_over/candidate_has_application_rejected_by_default_spec.rb

```diff
@@ -110,7 +110,9 @@ def then_i_see_my_application_is_now_unsuccessful
 
   def then_i_cannot_carry_over_my_application
     expect(page).to have_current_path candidate_interface_continuous_applications_choices_path
-    expect(page).to have_content("Applications for courses starting in September #{RecruitmentCycle.current_year} are closed.")
+    expect(page).to have_content(
+      "If your application(s) are not successful, or you do not accept any offers, you will be able to apply for courses starting in the #{CycleTimetable.cycle_year_range(RecruitmentCycle.next_year)} academic year from 8 October.",
```

Comment:
Wondering if 8 October will change across cycles

Reply:
Good catch, this was an oversight.